tests/unittest_brain_qt.py

```diff
@@ -28,11 +30,25 @@ def test_value_of_lambda_instance_attrs_is_list():
         from PyQt6 import QtPrintSupport as printsupport
         printsupport.QPrintPreviewDialog.paintRequested  #@
         """
-        AstroidManager.brain["extension_package_whitelist"] = {"PyQt6.QtPrintSupport"}
         node = extract_node(src)
         attribute_node = node.inferred()[0]
         if attribute_node is Uninferable:
             pytest.skip("PyQt6 C bindings may not be installed?")
         assert isinstance(attribute_node, UnboundMethod)
         # scoped_nodes.Lambda.instance_attrs is typed as Dict[str, List[NodeNG]]
         assert isinstance(attribute_node.instance_attrs["connect"][0], FunctionDef)
+
+    @staticmethod
+    def test_implicit_parameters():
```

Comment:
```suggestion
    def test_implicit_parameters() -> None:
```

Reply:
Happy to add it, but when I've seen this in the test suite, it's struck me as oddly specific. Why would a test method ever return anything? 😄

Reply:
I just like to add it as I think there can be a benefit to eventually turning on `mypy` even over the test suite. I think `mypy` actually caught some bugs for me in the `pylint` test suite already. Having the signature typed makes `mypy` actually check the function.

Reply:
> Having the signature typed makes mypy actually check the function.

Ah, thank you, this was the part that didn't come to mind!